terraform/github/repositories/ministryofjustice/cloud-optimisation-and-accountability.tf

```diff
@@ -0,0 +1,14 @@
+module "cloud-optimisation-and-accountability" {
+  source  = "ministryofjustice/repository/github"
+  version = "1.2.1"
+
+  poc = false
+
+  name        = "cloud-optimisation-and-accountability"
+  description = "A GitHub repository for the Cloud Optimisation and Accountability Team"
+  topics      = ["cloud-optimisation-and-accountability"]
+
+  team_access = {
+    admin = [var.cloud-optimisation-and-accountability_team.id]
```

Comment:
As far as I can see you haven't created this variable in the variable.tf file so you can't reference it (again it's looks like underscores are preferred/required)

`variable "cloud_optimisation_and_accountability_team_id" {
  description = "cloud-optimisation-and-accountability team id"
  type        = string
}`